search/rowrange.go

```diff
@@ -44,6 +44,57 @@ func intersectRowRanges(lhs, rhs []rowRange) []rowRange {
 			r++
 		}
 	}
+	return simplify(res)
+}
+
+// complement returns the complement of lhs inside rhs.
+// it assumes that lhs and rhs are simplified and returns a simplified result.
```

Comment:
What does it mean "it assumes that lhs and rhs are simplified"?

Reply:
They are result of the simplify function below

Reply:
Non on overlapping with strict monotonically increasing "from" value

Reply:
I see thanks!